DataFormats/Common/interface/WrapperBase.h

```diff
@@ -28,6 +28,7 @@ namespace edm {
     WrapperBase();
     ~WrapperBase() override;
     bool isPresent() const { return isPresent_(); }
+    void markAsPresent() { markAsPresent_(); }
```

Comment:
This is used by [`GenericCloner::produce()`](https://github.com/cms-sw/cmssw/pull/47504/files#diff-9161cd4c4ca38ce26494d87f6c52a5f9a6adf2304ecc8bc99ed7efac1114ccf6R145-R146) to mark a newly constructed `Wrapper<T>` as holding a valid object, before filling that object with `memcpy`.
Do you see any reasons not to have this method ?  Currently I cannot think of any way to avoid it, though.